query/mysql/mysql_test.go

```diff
@@ -76,13 +104,30 @@ func (s *TestSuite) TestExplainWithoutQuery(t *C) {
 }
 
 func (s *TestSuite) TestExplainWithoutDb(t *C) {
-	//TODO This test fails on MySQL 5.7+ because the response has new fields
```

Comment:
I've fixed the test so it now detects on which MySQL version it runs and tests accordingly.